packages/x-tailwindcss/src/x-tailwind-plugin/components/scroll/default.ts

```diff
@@ -0,0 +1,38 @@
+import { TailwindHelpers } from '../../../types';
+/**
+ * Returns the default styles for component `scroll`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function scrollDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    overflowY: 'scroll',
+    '--x-color-thumb-scroll-bar': theme('colors.neutral.25'),
+    '--x-color-thumb-scroll-bar-hover': theme('colors.neutral.50'),
+    '--x-color-background-scroll-bar': theme('colors.neutral.0'),
+
+    '@media (hover: hover)': {
```

Comment:
For some reason, the rules inside the `media` are repeated five times in the generated CSS.